### DISCUSSION TITLE
The application is not deployed on github pages

### DISCUSSION THREAD

**Discussion by AndreyZaytsev90** [community/community#143169] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![Снимок экрана от 2024-10-31 08-26-08](https://github.com/user-attachments/assets/10496f65-57ac-4ae9-a698-1944f6fc3bfe)
![Снимок экрана от 2024-10-31 08-29-05](https://github.com/user-attachments/assets/e3b5368a-eec5-4afa-9128-c9160f80f0cc)
![Снимок экрана от 2024-10-31 08-29-20](https://github.com/user-attachments/assets/31fd2dc5-9f7e-4afc-bdfc-db59c53306ca)
![Снимок экрана от 2024-10-31 08-25-54](https://github.com/user-attachments/assets/bca92efa-d98e-4338-b756-fc74c433c89d)
![Снимок экрана от 2024-10-31 08-26-28](https://github.com/user-attachments/assets/22eba786-0f3a-42dd-b37a-c36730e433e8)
![Снимок экрана от 2024-10-31 08-28-15](https://github.com/user-attachments/assets/6eaedf10-4d55-40b3-8161-e0c77d327c38)
![Снимок экрана от 2024-10-31 08-35-21](https://github.com/user-attachments/assets/a6690265-cb69-4c88-a901-3453afb7db55)

Build and deploy they pass without errors, but the new workflow does not appear and nothing happens

**Comment by AndreyZaytsev90** [comment]
Tell me what could be the problem? Maybe someone has faced a similar problem?

**Reply by LiteBrite82** [reply, marked as answer]
@AndreyZaytsev90 
It looks like it deployed. Are there further issues?
Side note: create-react-app has ben deprecated for a while now. Its better to use a different framework like "Vite" so you have better long term support.

**Reply by AndreyZaytsev90** [reply]
I have solved the problem. I had to take the time and switch to github actions. Now when I do commit and push everything happens automatically
The application is old, so it is still on CRA
I am already creating new applications on Vite
Thanks for the advice!
